library/sl

```diff
@@ -15,4 +15,4 @@ Tags: 6
 Architectures: amd64
 GitFetch: refs/heads/6
 Directory: sl6
-GitCommit: d2d17b81a784fc91219b6421f15080fb7205eb0d
+GitCommit: 2bf200b86a0d533576a1a49c93e50f4e65a5bc12
```

Comment:
Does SL 6 have changes in the base set of packages?  If the build is deterministic/reproducible, then this could be entirely expected if nothing has actually changed in what this is pointing to.

```console
$ md5sum \
    <(curl -fsSL 'https://github.com/scientificlinux/sl-docker/raw/d2d17b81a784fc91219b6421f15080fb7205eb0d/sl6/sl-6-docker.tar.xz') \
    <(curl -fsSL 'https://github.com/scientificlinux/sl-docker/raw/2bf200b86a0d533576a1a49c93e50f4e65a5bc12/sl6/sl-6-docker.tar.xz')
85b2d07e9325d51b3e6e2eafbf9c537b  /dev/fd/63
85b2d07e9325d51b3e6e2eafbf9c537b  /dev/fd/62
```

Reply:
It looks like there were not any base package changes in Feb, and the build is reproducible so I think this might be accurate.